src/http/api/account.ts

```diff
@@ -1,8 +1,15 @@
+import type { Account } from 'account/account.entity';
+import type { KnexAccountRepository } from 'account/account.repository.knex';
 import type { AccountService } from 'account/account.service';
 import { getAccountHandle } from 'account/utils';
-import type { AppContext } from 'app';
-import { sanitizeHtml } from 'helpers/html';
+import { type AppContext, fedify } from 'app';
+import { isHandle } from 'helpers/activitypub/actor';
+import { lookupAPIdByHandle } from 'lookup-helpers';
 import type { PostService } from 'post/post.service';
+import {
+    getAccountDTOByHandle,
+    getAccountDtoFromAccount,
```

Comment:
Any reason to use different casing on `DTO`?

Reply:
No reason. just a mistake